EXPLORER_RELEASE_SETUP_INSTRUCTIONS.md

```diff
@@ -0,0 +1,56 @@
+
+Before you begin, install Docker on your machine.  If you already have Docker installed, skip this step. Next, setup port forwarding for your storage node. Make sure you have an email with your personal single use authorization token. If you don’t have an authorization token yet, please join our waitlist. You will not be able to setup a staorage node if you dont have an authorization token.
+
+- To install `docker` please visit: [docker.com](https://docs.docker.com/install/) and follow the installation guide for your operating system. 
+- Set up port forwarding! Please visit our [knowledge base article](https://storjlabs.atlassian.net/wiki/spaces/SCKB/pages/edit/4423868?draftId=4292802&draftShareId=dc880538-dc43-4ad1-9691-425adaea7c5c&) or follow the instructions for your router on [portforward.com](https://portforward.com/).
+
+
+#### Setting up your Storage Node on the V3 Network!
+
+1) Download the Identity tool binary and create an Identity. The process of generating an identity could take several hours; it is dependent on your machine´s processing power & luck.
+
+	Download the correct binary for your operating system:
+	- Mac OS: [Darwin_amd64.zip](https://storj-v3-alpha-builds.storage.googleapis.com/a1027c7-go1.11/identity_darwin_amd64.zip )
+	- Linux: [Linux_amd64.zip](https://storj-v3-alpha-builds.storage.googleapis.com/a1027c7-go1.11/identity_linux_amd64.zip )
+	- Raspberry Pi: [Linux_arm.zip](https://storj-v3-alpha-builds.storage.googleapis.com/a1027c7-go1.11/identity_linux_arm.zip )
+	- Windows: [Windows_amd64.zip](https://storj-v3-alpha-builds.storage.googleapis.com/a1027c7-go1.11/identity_windows_amd64.zip )
+
+2) Unzip the file and run the following command to start creating an identity (this example is for Mac OS, substitute the appropriate identity binary for your OS):
+
+	`$ ./identity_darwin_amd64 create storagenode`
+
+3) Sign the identity you created with your personal single-use authorization token by running the following command: 
+
+	`$ identity authorize storagenode <authorization-token>`
+
+4) Download the docker container from docker hub: 
+
+	`$ docker pull storjlabs/storagenode:alpha`
+
+5) Run storage node with the following command, after editing `WALLET`, `EMAIL`, `ADDRESS`, and `<storage-dir>`
```

Comment:
Edit these parameters where?

Reply:
Once I looked down a bit I found the example command. Spoke to Brandon about this and to keep people from just trying to enter the command as is it might make sense to leave it where it is...